frontend/packages/ceph-storage-plugin/src/components/independent-mode/install.tsx

```diff
@@ -35,39 +34,11 @@ const InstallExternalCluster = withHandlePromise((props: InstallExternalClusterP
     match: {
       params: { ns, appName },
     },
+    minKeys,
```

Comment:
```suggestion
    requiredKeys,
```

Reply:
`minKeys` is better since we don't fail if there are extra keys.